apps/docs/components/blocks/Banners.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: DefaultLayout
+hideBreadcrumbs: true
+description: A breadcrumb trail consists of a list of links to the parent pages of the current page in hierarchical order. It helps users find their place.
```

Comment:
I think that description is from original file this one was copied from

Reply:
Thx, just forgot about changing.